packages/bigtable/src/index.js

```diff
@@ -262,267 +276,263 @@ function Bigtable(options) {
+      },
+      BigtableInstanceAdmin: {
+        baseUrl: 'bigtableadmin.googleapis.com',
+        path: googleProtoFiles(
+          'bigtable/admin/v2/bigtable_instance_admin.proto'
+        ),
+        service: 'bigtable.admin'
+      },
+      Operations: {
+        baseUrl: 'bigtableadmin.googleapis.com',
+        path: googleProtoFiles('longrunning/operations.proto'),
+        service: 'longrunning',
+        apiVersion: 'v1'
       }
     },
     scopes: [
       'https://www.googleapis.com/auth/bigtable.admin',
-      'https://www.googleapis.com/auth/bigtable.data'
+      'https://www.googleapis.com/auth/bigtable.data',
+      'https://www.googleapis.com/auth/cloud-platform'
     ],
     userAgent: PKG.name + '/' + PKG.version
   };
 
   common.GrpcService.call(this, config, options);
+
+  this.projectName = 'projects/' + this.projectId;
 }
 
 util.inherits(Bigtable, common.GrpcService);
 
 /**
- * Formats the full table name into a user friendly version.
- *
- * @private
- *
- * @param {string} name - The formatted Table name.
- * @return {string}
- *
- * @example
- * Bigtable.formatTableName_('projects/p/zones/z/clusters/c/tables/my-table');
- * // => 'my-table'
- */
-Bigtable.formatTableName_ = function(name) {
-  if (name.indexOf('/') === -1) {
-    return name;
-  }
-
-  var parts = name.split('/');
-  return parts[parts.length - 1];
-};
-
-/**
- * Create a table on your Bigtable cluster.
+ * Creates a Compute Instance.
  *
- * @resource [Designing Your Schema]{@link https://cloud.google.com/bigtable/docs/schema-design}
- * @resource [Splitting Keys]{@link https://cloud.google.com/bigtable/docs/managing-tables#splits}
+ * @resource [Creating a Compute Instance]{@link https://cloud.google.com/bigtable/docs/creating-compute-instance}
  *
- * @throws {error} If a name is not provided.
- *
- * @param {string} name - The name of the table.
- * @param {object=} options - Table creation options.
- * @param {object|string[]} options.families - Column families to be created
- *     within the table.
- * @param {string} options.operation - Operation used for table that has already
- *    been queued to be created.
- * @param {string[]} options.splits - Initial
- *    [split keys](https://cloud.google.com/bigtable/docs/managing-tables#splits).
+ * @param {string} name - The unique name of the instance.
+ * @param {object} options - Instance creation options.
+ * @param {string} options.displayName - The descriptive name for this instance
+ *     as it appears in UIs.
+ * @param {object[]} options.clusters - The clusters to be created within the
+ *     instance.
  * @param {function} callback - The callback function.
  * @param {?error} callback.err - An error returned while making this request.
- * @param {module:bigtable/table} callback.table - The newly created table.
+ * @param {module:bigtable/instance} callback.instance - The newly created
+ *     instance.
+ * @param {Operation} callback.operation - An operation object that can be used
```

Comment:
Nevermind, this is the grpc operation. We'll definitely need a way to document this so the user knows how it works.